metadata_layer.py

```diff
@@ -176,14 +176,16 @@ def __repr__(self):
 
 
 class ContributorData(object):
+
     def __init__(self, sort_name=None, display_name=None,
                  family_name=None, wikipedia_name=None, roles=None,
                  lc=None, viaf=None, biography=None, aliases=None, extra=None):
         self.sort_name = sort_name
         self.display_name = display_name
         self.family_name = family_name
         self.wikipedia_name = wikipedia_name
-        roles = roles or Contributor.AUTHOR_ROLE
+        if roles is None:
+            roles = Contributor.AUTHOR_ROLE
```

Comment:
This doesn't make a difference now, but I'm creating synthetic `ContributorData` objects representing individuals outside of whatever they may have contributed to a Work, and I want to make sure those objects don't have any roles associated with them.